Makefile

```diff
@@ -106,6 +106,7 @@ prereqs: ## Check if prerequisites are met, and install missing dependencies
 fmt: ## Run go fmt against code.
 	@echo "### Formatting code"
 	go fmt ./...
+	find ./bpf -type f -not -path "./bpf/headers/*" -name "*.[ch]" | xargs clang-format -i --Werror
```

Comment:
I have this rule as part of the lint target we can make it part of the fmt but in that case we need to remove it from the lint so we won't run it twice when build ?

Reply:
I think that's for two different things: the format rule allows to auto-format as part of the _local_ build process on your machine, whereas the lint rule is still useful _on the CI_ to make sure we didn't commit & push unformatted code.